README.md

```diff
@@ -0,0 +1,24 @@
+# Pipelines Installation Check
+
+This package demonstrates processing a single CCD with the Rubin Observatory
+science pipelines.
+
+## Usage
+
+Running the script `bin/run_demo.sh` will create a new data repository in the
+`DATA_REPO/` dir, ingest the required data from `export_dir`, and process them
```

Comment:
Being picky, `export_dir` seems like a weird name for this. I understand that it is exported from ci_hsc, but it's the primary *input* that most users of this package are going to be thinking about.